gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,226 @@
+            <li className="nx-list__item">
+              A navigation link section which display an icon plus text in the open state and an icon in the closed
+              state
+            </li>
+          </ul>
+          <NxP>
+            Because <NxCode>NxGlobalSidebar</NxCode> contains product branding it should not be used in conjunction
+            with <NxCode>NxPageHeader</NxCode> or <NxCode>NxNexusPageHeader</NxCode>. Instead it should be paired with
+            <NxCode>NxSystemHeader</NxCode> (not yet added to the gallery).
+          </NxP>
+          <NxP>
+            Addition custom content may be added below the navigation links if desired, care should be taken that such
+            content can adapt to both the open and closed states.
+          </NxP>
+          <section className="nx-tile-subsection">
+            <header className="nx-tile-subsection__header">
+              <h3 className="nx-h3"><NxCode>NxGlobalSidebar</NxCode> Props</h3>
+            </header>
+            <NxTable>
+              <NxTable.Head>
+                <NxTable.Row>
+                  <NxTable.Cell>Prop</NxTable.Cell>
+                  <NxTable.Cell>Type</NxTable.Cell>
+                  <NxTable.Cell>Required</NxTable.Cell>
+                  <NxTable.Cell>Details</NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Head>
+              <NxTable.Body>
+                <NxTable.Row>
+                  <NxTable.Cell>isOpen</NxTable.Cell>
+                  <NxTable.Cell>Boolean</NxTable.Cell>
+                  <NxTable.Cell>No</NxTable.Cell>
+                  <NxTable.Cell></NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>toggleOpenIcon</NxTable.Cell>
+                  <NxTable.Cell>IconDefinition</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>
+                    An icon used in the sidebar's open/close toggle to represent Opening the sidebar from a closed
+                    state.
+                  </NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>toggleClosedIcon</NxTable.Cell>
+                  <NxTable.Cell>IconDefinition</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>
+                    An icon used in the sidebar's open/close toggle to represent Opening the sidebar from an open
+                    state.
+                  </NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>logoImg</NxTable.Cell>
+                  <NxTable.Cell>string</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>
+                    An image placed in the top left corner which displays branding and product name.
```

Comment:
I believe there are requirements on the size and contents of the image which should be documented here.

Reply:
> I believe there are requirements on the size and contents of the image which should be documented here.

I cannot find any requirements in the mockups. Derrek?

Reply:
@dturner32 ^^

Reply:
> I cannot find any requirements in the mockups. Derrek?

I was referring to the aspect ratio and the fact that the image must include the text name of the product. This was discussed during the implementation ticket for the sidebar I believe.

Reply:
> > I cannot find any requirements in the mockups. Derrek?
> 
> I was referring to the aspect ratio and the fact that the image must include the text name of the product. This was discussed during the implementation ticket for the sidebar I believe.

That's fine, but it should still be documented in the mockups, especially the aspect ratio.